Source/Core/DolphinQt2/Settings/AudioPane.cpp

```diff
@@ -0,0 +1,227 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/AudioPane.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QRadioButton>
+#include <QSlider>
+#include <QSpinBox>
+#include <QVBoxLayout>
+
+#include "AudioCommon/AudioCommon.h"
+#include "DolphinQt2/Config/SettingsWindow.h"
+#include "DolphinQt2/Settings.h"
+
+AudioPane::AudioPane(SettingsWindow* parent)
+{
+  CreateWidgets();
+  LoadSettings();
+  ConnectWidgets();
+
+  connect(parent, &SettingsWindow::EmulationStarted, [this] { OnEmulationStateChanged(true); });
+  connect(parent, &SettingsWindow::EmulationStopped, [this] { OnEmulationStateChanged(false); });
+}
+
+void AudioPane::CreateWidgets()
+{
+  auto* dsp_box = new QGroupBox(tr("DSP Emulator Engine"));
+  auto* dsp_layout = new QVBoxLayout;
+  dsp_box->setLayout(dsp_layout);
+  m_dsp_hle = new QRadioButton(tr("DSP HLE emulation (fast)"));
+  m_dsp_lle = new QRadioButton(tr("DSP LLE recompiler"));
+  m_dsp_interpreter = new QRadioButton(tr("DSP LLE interpreter (slow)"));
+
+  dsp_layout->addWidget(m_dsp_hle);
+  dsp_layout->addWidget(m_dsp_lle);
+  dsp_layout->addWidget(m_dsp_interpreter);
+
+  dsp_box->setMaximumHeight(150);
+
+  auto* volume_box = new QGroupBox(tr("Volume"));
+  auto* volume_layout = new QGridLayout;
+  m_volume_slider = new QSlider;
+  m_volume_label = new QLabel(tr("??? %"));
+
+  volume_box->setLayout(volume_layout);
+  volume_box->setMaximumWidth(60);
+  volume_box->setMaximumHeight(150);
```

Comment:
These should likely be calculated from screen metrics if necessary (which the [Hi-DPI](https://doc.qt.io/qt-5/highdpi.html) guide recommends in migration; that is, not using fixed pixel widths). Fixed widths can make the UI look too small or too large depending on the scaling a display uses.